Dockerfile

```diff
@@ -33,6 +33,16 @@ COPY --from=builder /workspace/manager .
 COPY appdata.gitconfig /.gitconfig
 RUN chgrp -R 0 /.gitconfig && chmod -R g=u /.gitconfig
 
+# build tini and install to manage zombie proceses
+RUN git clone --branch v0.19.0 https://github.com/krallin/tini /tini
```

Comment:
Perhaps this should be moved to its own step? Otherwise we'll have to rebuild tini each time we modify HAS.

Reply:
yes, we will need to avoid rebuilding